outlook/src/taskpane/components/ProfileCard/ProfileCard.tsx

```diff
@@ -44,7 +44,7 @@ export class ProfileCard extends React.Component<ProfileCardProps, {}> {
 
         let iconOrInitials = <img className='icon' src={this.props.icon} onError={this.addDefaultSrc}/>;
 
-        if (this.props.icon) {
+        if (!this.props.icon) {
```

Comment:
Seems like a fix. Shouldn't it be in its own commit ?

Reply:
Hmm Yeah I think so.